### ISSUE TITLE
Fiona install fails with libpoppler error

### ISSUE THREAD

**Issue by jorisvandenbossche** [conda-forge/fiona-feedstock#62]
Our pandas CI where we test downstream packages including geopandas just started failing:

```
================================== FAILURES ===================================
________________________________ test_geopandas ________________________________
[gw1] linux -- Python 3.6.3 /home/travis/miniconda3/envs/pandas/bin/python
    def test_geopandas():
    
>       geopandas = import_module('geopandas')  # noqa
pandas/tests/test_downstream.py:97: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pandas/tests/test_downstream.py:19: in import_module
    return importlib.import_module(name)
../../../miniconda3/envs/pandas/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:994: in _gcd_import
    ???
<frozen importlib._bootstrap>:971: in _find_and_load
   ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:665: in _load_unlocked
    ???
<frozen importlib._bootstrap_external>:678: in exec_module
    ???
<frozen importlib._bootstrap>:219: in _call_with_frames_removed
    ???
../../../miniconda3/envs/pandas/lib/python3.6/site-packages/geopandas/__init__.py:4: in <module>
    from geopandas.io.file import read_file
../../../miniconda3/envs/pandas/lib/python3.6/site-packages/geopandas/io/file.py:3: in <module>
    import fiona
../../../miniconda3/envs/pandas/lib/python3.6/site-packages/fiona/__init__.py:69: in <module>
    from fiona.collection import Collection, BytesCollection, vsi_path
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    import logging
    import os
    import warnings
    
    from fiona import compat
>   from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
E   ImportError: libpoppler.so.66: cannot open shared object file: No such file or directory
../../../miniconda3/envs/pandas/lib/python3.6/site-packages/fiona/collection.py:9: ImportError
```

The logs are here: https://travis-ci.org/pandas-dev/pandas/jobs/306469511#L2019. You need to expand line 541 "$ ci/install_travis.sh" to see the install logs. As far as I can see, everything is coming from conda-forge, and there are no downgrades/changes that could mess things up after fiona and geopandas are installed.

**Comment by jorisvandenbossche** [comment]
I can replicate this locally in a clean env:

```
joris@joris-XPS-13-9350:~/scipy$ conda create -n TEST python=3 geopandas -c conda-forge
Fetching package metadata ...............
Solving package specifications: .

Package plan for installation in environment /home/joris/miniconda3/envs/TEST:

The following NEW packages will be INSTALLED:

    backports:                     1.0-py36_1            conda-forge
    backports.functools_lru_cache: 1.4-py36_1            conda-forge
    ca-certificates:               2017.11.5-0           conda-forge
    cairo:                         1.14.6-5              conda-forge
    certifi:                       2017.11.5-py36_0      conda-forge
    click:                         6.7-py36_0            conda-forge
    click-plugins:                 1.0.3-py36_0          conda-forge
    cligj:                         0.4.0-py36_0          conda-forge
    curl:                          7.55.1-0              conda-forge
    cycler:                        0.10.0-py36_0         conda-forge
    dbus:                          1.10.22-0             conda-forge
    descartes:                     1.1.0-py36_0          conda-forge
    expat:                         2.1.0-3               conda-forge
    fiona:                         1.7.9-py36_0          conda-forge
    fontconfig:                    2.12.1-6              conda-forge
    freetype:                      2.7-2                 conda-forge
    freexl:                        1.0.4-0               conda-forge
    geopandas:                     0.3.0-py36_0          conda-forge
    geos:                          3.6.2-1               conda-forge
    geotiff:                       1.4.2-0               conda-forge
    gettext:                       0.19.7-1              conda-forge
    giflib:                        5.1.4-0               conda-forge
    glib:                          2.51.4-0              conda-forge
    gst-plugins-base:              1.8.0-0               conda-forge
    gstreamer:                     1.8.0-2               conda-forge
    hdf4:                          4.2.13-0              conda-forge
    hdf5:                          1.10.1-1              conda-forge
    icu:                           58.2-0                conda-forge
    intel-openmp:                  2018.0.0-hc7b2577_8              
    jpeg:                          9b-2                  conda-forge
    json-c:                        0.12.1-0              conda-forge
    kealib:                        1.4.7-4               conda-forge
    krb5:                          1.14.2-0              conda-forge
    libdap4:                       3.18.3-2              conda-forge
    libffi:                        3.2.1-3               conda-forge
    libgcc-ng:                     7.2.0-h7cc24e2_2                 
    libgdal:                       2.1.4-6               conda-forge
    libgfortran:                   3.0.0-1                          
    libgfortran-ng:                7.2.0-h9f7466a_2                 
    libiconv:                      1.15-0                conda-forge
    libkml:                        1.3.0-3               conda-forge
    libnetcdf:                     4.4.1.1-10            conda-forge
    libpng:                        1.6.28-2              conda-forge
    libpq:                         9.6.3-0               conda-forge
    libspatialindex:               1.8.5-1               conda-forge
    libspatialite:                 4.3.0a-17             conda-forge
    libssh2:                       1.8.0-2               conda-forge
    libstdcxx-ng:                  7.2.0-h7a57d05_2                 
    libtiff:                       4.0.7-1               conda-forge
    libxcb:                        1.12-1                conda-forge
    libxml2:                       2.9.5-2               conda-forge
    matplotlib:                    2.1.0-py36_1          conda-forge
    mkl:                           2018.0.1-h19d6760_4              
    munch:                         2.2.0-py36_0          conda-forge
    ncurses:                       5.9-10                conda-forge
    numpy:                         1.13.3-py36ha12f23b_0            
    openjpeg:                      2.3.0-1               conda-forge
    openssl:                       1.0.2l-0              conda-forge
    pandas:                        0.21.0-py36_0         conda-forge
    pcre:                          8.39-0                conda-forge
    pixman:                        0.34.0-0              conda-forge
    poppler:                       0.61.1-0              conda-forge
    poppler-data:                  0.4.8-0               conda-forge
    proj4:                         4.9.3-5               conda-forge
    psycopg2:                      2.7.3.2-py36_0        conda-forge
    pyparsing:                     2.2.0-py36_0          conda-forge
    pyproj:                        1.9.5.1-py36_0        conda-forge
    pyqt:                          5.6.0-py36_4          conda-forge
    pysal:                         1.14.3-py36_0         conda-forge
    python:                        3.6.3-1               conda-forge
    python-dateutil:               2.6.1-py36_0          conda-forge
    pytz:                          2017.3-py_2           conda-forge
    qt:                            5.6.2-6               conda-forge
    readline:                      6.2-0                 conda-forge
    rtree:                         0.8.3-py36_0          conda-forge
    scipy:                         1.0.0-py36hbf646e7_0             
    setuptools:                    36.7.2-py36_0         conda-forge
    shapely:                       1.6.2-py36_1          conda-forge
    sip:                           4.18-py36_1           conda-forge
    six:                           1.11.0-py36_1         conda-forge
    sqlalchemy:                    1.1.13-py36_0         conda-forge
    sqlite:                        3.13.0-1              conda-forge
    tk:                            8.5.19-2              conda-forge
    tornado:                       4.5.2-py36_0          conda-forge
    util-linux:                    2.21-0                           
    xerces-c:                      3.1.4-3               conda-forge
    xorg-libxau:                   1.0.8-3               conda-forge
    xorg-libxdmcp:                 1.1.2-3               conda-forge
    xz:                            5.2.3-0               conda-forge
    zlib:                          1.2.11-0              conda-forge

Proceed ([y]/n)? 

intel-openmp-2 100% |##############################################################################################################################################################################################| Time: 0:00:02 227.23 kB/s
libiconv-1.15- 100% |##############################################################################################################################################################################################| Time: 0:00:08 235.49 kB/s
poppler-data-0 100% |##############################################################################################################################################################################################| Time: 0:00:15 237.41 kB/s
freexl-1.0.4-0 100% |##############################################################################################################################################################################################| Time: 0:00:00 253.75 kB/s
libxml2-2.9.5- 100% |##############################################################################################################################################################################################| Time: 0:00:21 237.63 kB/s
mkl-2018.0.1-h 100% |##############################################################################################################################################################################################| Time: 0:16:48 192.07 kB/s
openjpeg-2.3.0 100% |##############################################################################################################################################################################################| Time: 0:00:01 222.28 kB/s
fontconfig-2.1 100% |##############################################################################################################################################################################################| Time: 0:00:04 220.86 kB/s
poppler-0.61.1 100% |##############################################################################################################################################################################################| Time: 0:00:28 236.84 kB/s
#
# To activate this environment, use:
# > source activate TEST
#
# To deactivate an active environment, use:
# > source deactivate
#

joris@joris-XPS-13-9350:~/scipy$ act TEST
(TEST) joris@joris-XPS-13-9350:~/scipy$ python
Python 3.6.3 | packaged by conda-forge | (default, Nov  4 2017, 10:10:56) 
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import geopandas
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/joris/miniconda3/envs/TEST/lib/python3.6/site-packages/geopandas/__init__.py", line 4, in <module>
    from geopandas.io.file import read_file
  File "/home/joris/miniconda3/envs/TEST/lib/python3.6/site-packages/geopandas/io/file.py", line 3, in <module>
    import fiona
  File "/home/joris/miniconda3/envs/TEST/lib/python3.6/site-packages/fiona/__init__.py", line 69, in <module>
    from fiona.collection import Collection, BytesCollection, vsi_path
  File "/home/joris/miniconda3/envs/TEST/lib/python3.6/site-packages/fiona/collection.py", line 9, in <module>
    from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
ImportError: libpoppler.so.66: cannot open shared object file: No such file or directory
```

**Comment by ocefpaf** [comment]
Thanks for the report! This will be solved as soon as https://github.com/conda-forge/poppler-feedstock/pull/8 and https://github.com/conda-forge/libgdal-feedstock/pull/24 are merged. Meanwhile you can pin to the previous version of `popppler 0.52.0` by specifying it explicitly on the env.

**Comment by ocefpaf** [comment]
@jorisvandenbossche I merged https://github.com/conda-forge/libgdal-feedstock/pull/24 and just tested the env above. Everything should be working. Let me know if you find any issue.

**Comment by jorisvandenbossche** [comment]
Thanks! CI is already running to see if it is solved: https://github.com/pandas-dev/pandas/pull/18466, but on the geopandas docs it now works, so I assume everything is working again

**Comment by jorisvandenbossche** [comment]
Our CI is green again!

**Comment by ocefpaf** [comment]
> Our CI is green again!

Awesome!

**Comment by jorisvandenbossche** [comment]
New issue with libxerces-c, see eg https://travis-ci.org/pandas-dev/pandas/jobs/306830766

**Comment by ocefpaf** [comment]
Should be gone soon. I thought I'd be the only one working on the Thanksgiving weekend but I guess I forgot about people in the old continent :wink: 

I am updating a bunch of stuff and you are hitting the Travis-CI backlog. The builds with latest xerces-c should be in the channel in ~ 30 min.

**Comment by jorisvandenbossche** [comment]
Good to hear!

(Yes, here it was just a normal working day ;-))

**Comment by ocefpaf** [comment]
@jorisvandenbossche it should be OK now. Let me know if you see any other errors.

PS: you'll probably get `gdal 2.1.3` for now. Once https://github.com/conda-forge/gdal-feedstock/pull/185 is merged you'll get `gdal 2.1.4`.
